src/site/outline/outline.rst

```diff
@@ -110,63 +91,58 @@ Below is a list of topics in no particular order that may or may not be covered
 * Sorting Algorithms
 
 
-
 Student Evaluation (Tentative Dates)
 ====================================
 
 +------------------------+------------+---------------------+
```

Comment:
Worth switching to [list table](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/tables.html#list-table-directive), like many of the others?

Reply:
This was on the todo list for allll tables in the course. This, along with changing "images" to "figures", because they're better. 

I will leave it as is here, but will make an issue for bookkeeping.